### PR TITLE
Remove package listings

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @Eilon - we don't consistently list packages in most repos. I'd prefer removing this (or limiting it to listing the version in nuget.org). Thoughts?

**Comment by Eilon** [comment]
@DamianEdwards wanted this. Why remove it?

**Comment by pranavkm** [comment]
Well, we should add it to all the repos in that case (and update the feeds when we migrate).. Right now, there are 4 repos under aspnet which have feed details.

**Comment by Eilon** [comment]
Do the shields.io images work with dotnet.myget.org? But anyway, I don't think this needs to be an all-or-none.

**Comment by pranavkm** [comment]
Fair enough. Regardless, we shouldn't point people to untested bits. Could we change this to just the signed \ tested build + nuget.org?

**Comment by Eilon** [comment]
Yeah I agree there are a few to many package variations listed here. Feel free to trim as needed.

**Comment by pranavkm** [comment]
Updated

File: README.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ AppVeyor: [![AppVeyor](https://ci.appveyor.com/api/projects/status/omn0l2l3mfhfj
 
 Travis:   [![Travis](https://travis-ci.org/aspnet/Localization.svg?branch=dev)](https://travis-ci.org/aspnet/Localization)
 
-| Package | aspnetvolatiledev | [aspnetcidev](https://myget.org/gallery/aspnetcidev) | [aspnetvnext](https://myget.org/gallery/aspnetvnext) | [NuGet](https://nuget.org) |
+| Package | [aspnet-core](https://myget.org/gallery/aspnetcore-dev) | [NuGet](https://nuget.org) |
```
Comment (Eilon) [inline README.md]: dotnet.myget.org?

**Comment by Eilon** [comment]
One comment, then :shipit:

**Comment by pranavkm** [comment]
Merged in https://github.com/aspnet/Localization/commit/09123cb349c03c497cff10e5105f87e18da64968

**Pull request closed without merging**